src/portable/portable.c

```diff
@@ -1559,40 +1559,46 @@ int portable_detach(
         }
 
         FOREACH_DIRENT(de, d, return log_debug_errno(errno, "Failed to enumerate '%s' directory: %m", where)) {
-                _cleanup_free_ char *marker = NULL;
-                UnitFileState state;
+                _cleanup_free_ char *marker = NULL, *unit_name = NULL;
+                const char *dot;
+
+                /* "portablectl --enable detach" calls DisableUnitFilesWithFlags DBus method, and the main
```

Comment:
This needs to specify "if it is symlinked via --copy=symlink", as that's the condition that triggers it

Reply:
@yuwata this was not addressed?

Reply:
Ouch, I mistakenly mention --runtime instead of --copy. Will post a follow-up.

Reply:
#23571.